fairlearn/preprocessing/_correlation_remover.py

```diff
@@ -145,16 +159,21 @@ def _check_sensitive_features_in_X(self, X) -> None:
             missing_columns = [i for i in self.sensitive_feature_ids if i not in range(X.shape[1])]
 
         if len(missing_columns) > 0:
-            raise ValueError(f"Columns {missing_columns} not found in the input data.")
+            raise ValueError(
+                "0 feature(s) (shape=(%d, 0)) while a minimum of %d is required. "
+                "Columns %s not found in the input data."
+                % (len(missing_columns), len(missing_columns), missing_columns)
+            )
 
     def _more_tags(self):
         return {
             "_xfail_checks": {
                 "check_transformer_data_not_an_array": (
                     "this estimator only accepts pandas dataframes or numpy ndarray as input."
                 ),
-                "check_estimators_empty_data_messages": (
-                    "this estimator raises on missing sensitive features in data."
-                ),
             }
         }
+
+    def __sklearn_tags__(self):
+        tags = super().__sklearn_tags__()
+        return tags
```

Comment:
With the new `parametrize_with_checks` you don't need to have this information here anymore so they can be removed

Reply:
I didn't not read with such attention the example code and that is on ME :D done in https://github.com/fairlearn/fairlearn/pull/1451/commits/25e647dfca52f9453daee75cde933aedf7762ef8